src/iGenVar.cpp

```diff
@@ -13,116 +13,163 @@
+    parser.add_option(args.vcf_sample_name, sharg::config{
+                        .short_id = 's', .long_id = "vcf_sample_name",
+                        .description = "Specify your sample name for the vcf header line.",
+                        .advanced = false});
 
     // Options - Other parameters:
-    parser.add_option(args.threads, 't', "threads",
-                      "Specify the number of decompression threads used for reading BAM files.",
-                      seqan3::option_spec::standard);
-    parser.add_flag(gVerbose, 'v', "verbose",
-                    "If you set this flag, we provide additional details about what iGenVar does. The detailed output "
-                    "is printed in the standard error.");
+    parser.add_option(args.threads, sharg::config{
+                        .short_id = 't', .long_id = "threads",
+                        .description = "Specify the number of decompression threads used for reading BAM files.",
+                        .advanced = false});
+    parser.add_flag(gVerbose, sharg::config{
+                        .short_id = 'v', .long_id = "verbose",
+                        .description = "If you set this flag, we provide additional details about what iGenVar does. "
+                                       "The detailed output is printed in the standard error."});
 
     // Options - Optional output:
-    parser.add_option(args.junctions_file_path, 'a', "junctions",
-                      "The path of the optional junction output file. If no path is given, junctions will not be output.",
-                      seqan3::option_spec::advanced,
-                      seqan3::output_file_validator{seqan3::output_file_open_options::open_or_create});
-    parser.add_option(args.clusters_file_path, 'b', "clusters",
-                      "The path of the optional cluster output file. If no path is given, clusters will not be output.",
-                      seqan3::option_spec::advanced,
-                      seqan3::output_file_validator{seqan3::output_file_open_options::open_or_create});
+    parser.add_option(args.junctions_file_path, sharg::config{
+                        .short_id = 'a', .long_id = "junctions",
+                        .description = "The path of the optional junction output file. If no path is given, junctions will not be output.",
+                        .advanced = true,
+                        .validator = sharg::output_file_validator{sharg::output_file_open_options::open_or_create}});
+    parser.add_option(args.clusters_file_path, sharg::config{
+                        .short_id = 'b', .long_id = "clusters",
+                        .description = "The path of the optional cluster output file. If no path is given, clusters will not be output.",
+                        .advanced = true,
+                        .validator = sharg::output_file_validator{sharg::output_file_open_options::open_or_create}});
 
     // Options - Methods:
```

Comment:
In general, do you know of this:

```
parser.add_subsection("Methods options");
```
Could maybe add more structure to the many options?
also takes a second bolean parameter whether the section name is only displayed at the advanced help page